x-pack/test_serverless/functional/test_suites/observability/cypress/cypress.config.ts

```diff
@@ -30,4 +39,9 @@ export default defineCypressConfig({
     supportFile: './support/e2e.ts',
     specPattern: './e2e/**/*.cy.ts',
   },
+  env: {
+    username: kbnTestConfig.getUrlParts().username,
+    password: kbnTestConfig.getUrlParts().password,
```

Comment:
What's the advantage of having this as env variables? The downside I see is that we'll probably want to test with other users in the near future. Do we also add those to env?

Reply:
We cannot use the package in the tests, something used inside the package it's colliding with cypress. This is what I see whenever I try to import the package within cypress context
<img width="1654" alt="image" src="https://github.com/elastic/kibana/assets/1313018/85b31b31-f44b-4a51-8601-e9b87f3b85e8">

Reply:
Ah.. It's because the `kbnTestConfig` package is server-side and you are trying to load it in the browser. There is not much we can do about that so I agree that env vars is the best solution for now.